pom.xml

```diff
@@ -116,6 +116,16 @@
                 <artifactId>worker-markup-testing</artifactId>
                 <version>3.3.0-SNAPSHOT</version>
             </dependency>
+            <dependency>
+                <groupId>com.github.workerframework.testing</groupId>
+                <artifactId>workerframework-testing-util</artifactId>
+                <version>1.0.0-SNAPSHOT</version>
+            </dependency>
+            <dependency>
+                <groupId>com.github.workerframework.testing</groupId>
+                <artifactId>workerframework-testing-integration</artifactId>
```

Comment:
Dependency management should be in alphabetical order.

Reply:
re-ordered